packages/react-magma-dom/src/components/Datagrid/Datagrid.test.js

```diff
@@ -836,6 +841,18 @@ describe('Datagrid', () => {
     });
   });
 
+  it('should have a title above the Datagrid', () => {
```

Comment:
`should display the title` is more accurate to what the test is doing (we aren't checking position, just that it's present